components/mail/evolution/Makefile

```diff
@@ -25,11 +25,11 @@ COMPONENT_ARCHIVE_HASH=	\
     sha256:33ab38780e9abb71506230e331e9c3534444b5ed3f17046f0b6371a00df686fb
 COMPONENT_ARCHIVE_URL=	http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/2.32/$(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.bz2
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/configure.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin:/usr/perl5/bin
+PATH=$(PATH.gnu):/usr/perl5/bin
```

Comment:
I think you can omit :/usr/perl5/bin because PATH:gnu already includes $(PERL5BINDIR).

Reply:
Fixed.